kubernetes/linux/main.sh

```diff
@@ -972,7 +972,7 @@ if [ ! -e "/etc/config/kube.conf" ]; then
             fi
             echo "using fluentbitconf file: ${fluentBitConfFile} for fluent-bit"
             if [ "$CONTAINER_RUNTIME" == "docker" ]; then
-                  fluent-bit -c /etc/opt/microsoft/docker-cimprov/${fluentBitConfFile}-e /opt/fluent-bit/bin/out_oms.so &
+                  fluent-bit -c /etc/opt/microsoft/docker-cimprov/${fluentBitConfFile} -e /opt/fluent-bit/bin/out_oms.so &
```

Comment:
we should be using linters/static check tools to avoid these. shellcheck is a good one for bash and is available in most of the IDEs

Reply:
> we should be using linters/static check tools to avoid these. shellcheck is a good one for bash and is available in most of the IDEs

@pfrcks , shellecheck seems good tool we should integrate. But, I dont see this being caught by tool. Are you able to catch this by any linter tool?

Reply:
Will create a task for the same

Reply:
you are right this won't be caught by shellcheck since this is a valid string in bash :(